src/main/java/com/app/library/constant/BorrowStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.app.library.constant;
+
+public enum BorrowStatus {
+    BORROWED,
```

Comment:
Move BorrowStatus and UserRole classes to model/enums package ;)

Reply:
Done